### DISCUSSION TITLE
Improved documentation on Nullable Lists

### DISCUSSION THREAD

**Discussion by RoboZoom** [MichalLytek/type-graphql#981] [Ideas]
From the documentation, I was having trouble understanding the settings I needed to use to get the right level of nullability in my schema when I have `nullableByDefault` set to true.  Specifically, I found that paradoxically setting nullable to `true` on a list changes the schema from `myField: [string]` to `myField: [string!]`

I understand *why* this is, with the nullable setting for `itemsAndLists`, but I think the documentation could use some improvement to help folks like me get to the right answer more quickly.  I'm happy to write up some proposed language for the documentation and submit a pull request, but looking through the docs in the `1.1.1` folder, I couldn't find the `Types and Fields` page.  

If someone can help point me in the right direction I'm happy to help out here.

**Comment by MichalLytek** [comment]
> looking through the docs in the 1.1.1 folder, I couldn't find the Types and Fields page.

You should modify the markdown file in `docs` folder directly. Then we can try to backport it to previous version of docs.

**Reply by RoboZoom** [reply]
Sounds good - I'll submit a PR in the next 1-2 days.
